modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -284,9 +279,7 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
          */
         virtual Ptr<BackendNode> initHalide(const std::vector<Ptr<BackendWrapper> > &inputs);
 
-        virtual Ptr<BackendNode> initInfEngine(const std::vector<Ptr<BackendWrapper> > &inputs);
-
-        virtual Ptr<BackendNode> initNgraph(const std::vector<Ptr<BackendWrapper> > &inputs, const std::vector<Ptr<BackendNode> >& nodes);
+        virtual Ptr<BackendNode> initNgraph(const std::vector<Ptr<BackendNode> >& nodes);
```

Comment:
What do you think, it's better to rename it to initial `initInfEngine`? But let's discuss before change :)